xray/audit/java/gradle.go

```diff
@@ -40,10 +48,45 @@ func runGradle(buildConfiguration *utils.BuildConfiguration, excludeTestDeps, us
 			log.Debug("Using resolver config from", configFilePath)
 		}
 	}
+	// check if gradle wrapper exist
+	if useWrapper {
+		var wrapperExist bool
+		wrapperExist, err = verifyGradleWrapper()
+		if err != nil {
+			return
+		}
+		useWrapper = wrapperExist
+	}
 	// Read config
 	vConfig, err := utils.ReadGradleConfig(configFilePath, useWrapper)
 	if err != nil {
 		return err
 	}
 	return gradleutils.RunGradle(vConfig, tasks, "", buildConfiguration, 0, useWrapper, true)
 }
+
+func verifyGradleWrapper() (bool, error) {
```

Comment:
IMO "verify" returns an error if the wrapper doesn't exist.
```suggestion
func isGradleWrapperExist() (bool, error) {
```

Reply:
done